Domains/4-Application/OpenSite/Released/OpenSiteDraft.01.00.00.ecschema.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="OpenSiteDraft" alias="OpenSiteDraft" version="01.00.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="OpenSite Draft" description="Drafts schema for drawing precise shapes in OpenSite+.">
```

Comment:
The version in the XML is 1.0.0 but the filename indicates 1.0.2. They should match.

BTW, also please keep the WIP version one ahead of the released one.